documentation/book/con-custom-resources-example.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-custom-resources.adoc
+
+[id='con-custom-resources-example-{context}']
+= {Productname} custom resource example
+Each {Productname}-specific custom resource is defined within the specification of a CRD.
+
+To understand the relationship between a CRD and a custom resource, let's look at a sample of the CRD for a Kafka topic.
+
+.Kafka topic CRD
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: CustomResourceDefinition
+Metadata: <1>
+  name: kafkatopics.kafka.strimzi.io
+  labels:
+    app: strimzi
+Spec: <2>
+  group: kafka.strimzi.io
+  versions:
+    {ApiVersion}
+  scope: Namespaced
+  names:
+    # ...
+    plural: kafkatopics
+    shortNames:
+    - kt <3>
+  additionalPrinterColumns: <4>
+      # ...
+  Validation: <5>
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          properties:
+            partitions:
+              type: integer
+              minimum: 1
+            replicas:
+              type: integer
+              minimum: 1
+              maximum: 32767
+      # ...
+----
+<1> The metadata for the topic custom resource, its name and a label to identify the resource.
+<2> The specification for the topic, including the group (domain) name, the plural name and the supported schema version, which are used in the URL to access the API of the topic. The other names are used to identify the resource in the CLI.
+<3> The shortname can be used in CLI commands, so `get kt` is used instead of `get kafka topic`.
+<4> The information presented when using a `get` command on the custom resource.
+<5> openAPIV3Schema validation provides validation for the creation of topic custom resources. For example, a topic requires at least one partition and one replica.
+
+NOTE: You can identify the CRD YAML files supplied with the {Productname} installation files, as the file names contain an index number followed by ‘Crd’.
+
+Here is a corresponding example of a `KafkaTopic` custom resource.
+
+.Kafka topic custom resource
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaTopic
```

Comment:
Callout that together the `kind` and `apiVersion` identify the CRD of which this CR is an instance.

Reply:
New callout added.